Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Mech/Weapons/Melee/debug.yml

```
@@ -0,0 +1,18 @@
+- type: entity
+  id: WeaponMechDebugMelle
+  parent: [ BaseMechWeaponMelee, DebugMechEquipment ] # Debug equipment does have all whitelist tags.
+  name: debug bam
+  suffix: Mech Weapon, DEBUG, Melee
+  description: A robust thing.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)